wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/registry/LanguageServerRegistryImpl.java

```diff
@@ -10,6 +10,17 @@
  *******************************************************************************/
 package org.eclipse.che.api.languageserver.registry;
 
+import static org.eclipse.che.api.languageserver.shared.ProjectLangugageKey.createProjectKey;
```

Comment:
The order or imports is not expected
https://github.com/codenvy-legacy/dev/wiki/Source-file#imports